nextcloud-startup-script.sh

```diff
@@ -476,6 +476,8 @@ then
     echo
     CONTEST=$(bash $SCRIPTS/test_connection.sh)
     if [ "$CONTEST" == "Connected!" ] then
+        echo "Connected!"
```

Comment:
Added this else they don't get confirmation its really connected opposed to "not connected" and its looks bad without.